src/main/java/ch/njol/skript/effects/EffInvulnerability.java

```diff
@@ -40,8 +40,8 @@ public class EffInvulnerability extends Effect {
 	
 	static {
 		Skript.registerEffect(EffInvulnerability.class,
-			"make %entities% invulnerable",
-			"make %entities% (not invulnerable|vulnerable)");
+			"make %entities% (invulnerable|invincible)",
+			"make %entities% (not (invulnerable|invincible)|vulnerable)");
```

Comment:
I think `vincible` works here too right?
```suggestion
			"make %entities% (not (invulnerable|invincible)|vulnerable|vincible)");
```

Reply:
is that a real word?

Reply:
I searched it up on Google, look like it 😬

Reply:
i mean sure i guess i could add it
I doubt anyone will ever use it though haha